gradle.properties

```diff
@@ -7,14 +7,14 @@ pluginRepositoryUrl=https://github.com/bufbuild/intellij-buf
 pluginVersion=0.2.1
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild=223
+pluginSinceBuild=233
```

Comment:
Do we need to bump to the latest? We had been supporting several older versions so far.

While we're in here, we should bump `pluginVersion` above to 0.3.0.

Reply:
I have checked compatibility with devkanro's protobuf plugin 1.6.32(target the 2022.3)